### PR TITLE
fix(graphql): add mozilla/fxa-graphql-api docker container

### PR DESCRIPTION
**Pull request by jrgm**
So, for a stack name `fxadev-jrgm`, graphql playground is available as https://graphql-jrgm.dev.lcip.org/graphql

Apologies for also including the cleanup I did to stop pulling defunct fxa-oauth server. Can see the change separately in https://github.com/mozilla/fxa-dev/commit/fe86459e9abce7d776dff3dd1a6910b92229da60 

r? - @dannycoates or @vladikoff /cc @bbangert

### CODE REVIEW

File: aws/environments/jrgm.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ ec2_volume_size: 24
 owner: "dev-fxacct@mozilla.org"
 reaper_spare_me: "true"
 
-fxadev_git_version: docker-jrgm-fix-auth-oauth
+fxadev_git_version: jrgm-fxa-graphql-api
+#graphql_api_docker_tag: dockerpush.graphql-api-dbg
```
Comment (jrgm) [inline aws/environments/jrgm.yml:15]: Ignore this file. Just used to build this branch.

File: aws/local.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@
     - verifier
```
Comment (jrgm) [inline aws/local.yml:26]: The oauth role is still here to do some config, which really should be moved fully into the auth role, but not this round.

File: roles/graphql-api/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      NODE_ENV: "development"
+      AUTH_SERVER_URL: "{{ auth_public_url }}"
+      PROFILE_SERVER_URL: "{{ profile_public_url }}"
+      ACCESS_TOKEN_REDIS_HOST: 127.0.0.1
+      SENTRY_DSN: ''
+      AUTH_MYSQL_DATABASE: fxa
+      AUTH_MYSQL_HOST: 127.0.0.1
+      AUTH_MYSQL_USERNAME: root
+      AUTH_MYSQL_PASSWORD: ''
+      OAUTH_MYSQL_DATABASE: fxa_oauth
```
Comment (jrgm) [inline roles/graphql-api/tasks/main.yml:45]: Oops. Forgot to use vars for these config items (so it works when fxa-dev uses a remote rds database). I'll work on that change now.

Comment (jrgm) [reply roles/graphql-api/tasks/main.yml:45]: updated to use variables

**Review by dannycoates — APPROVED** [review]
LGTM

**Pull request merged**